chapter_12_organising_test_files.asciidoc

```diff
@@ -602,8 +600,6 @@ and that can be executed later, and multiple times:
 
 Let's see our funky `wait_for` helper in action:
 
-// DAVID: Might be worth explicitly telling the reader to add that code earlier - I hadn't yet,
```

Comment:
I think the previous few paragraphs (ch11l010) were clear about implementing this. Has anyone else find it unclear though? If so, we can try making it clearer.